src/ImageGallery.jsx

```diff
@@ -301,7 +318,8 @@ export default class ImageGallery extends React.Component {
   }
 
   componentWillUnmount() {
-    window.removeEventListener('keydown', this.handleKeyDown);
+    const { useWindowKeyDown } = this.props
+    this.imageGallery.current.removeEventListener('keydown', this.handleKeyDown);
```

Comment:
This looks like its already handled below on line 336.

Reply:
@xiaolin Fixed! Let me know if you need anything else!